src/domains/checkout-common/03_redis.tf

```diff
@@ -53,3 +57,130 @@ module "pagopa_proxy_redis" {
 
   tags = var.tags
 }
+
+# -----------------------------------------------
+# end of pagopa checkout Redis configuration
+# -----------------------------------------------
+
+
+# -----------------------------------------------
+# start of pagopa checkout Redis configuration
+# -----------------------------------------------
+
+# Redis resource group
+resource "azurerm_resource_group" "redis_checkout_rg" {
+  name     = "${local.project}-redis-rg"
+  location = var.location
+  tags     = var.tags
+}
+
+# Redis cache subnet
+module "pagopa_checkout_redis_snet" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//subnet?ref=v8.83.1"
+
+  name                                      = format("%s-redis-snet", local.project)
```

Comment:
You can use interpolation as it is used in the definition of the resource group

Reply:
fixed with https://github.com/pagopa/pagopa-infra/pull/2804/commits/018e75896345fd1feb4e22e34c577ef837fa310b